Makefile

```
@@ -233,7 +233,7 @@ release-prep:
 	@rm -rf release_deps
 	@mkdir release_deps
 	@echo 'v$(CURR_VERSION)' > release_deps/VERSION.txt
-	./buildscripts/download-dependencies.sh release_deps
+	bash ./buildscripts/download-dependencies.sh release_deps
```

Can we use just sh here if necessary?
```suggestion
	sh ./buildscripts/download-dependencies.sh release_deps
```

---

We need to use bash for list support, we can revisit making this script work with posix shells in a future PR.